### DISCUSSION TITLE
10% Fee For Organizations To Sponsor Is Outrageous

### DISCUSSION THREAD

**Discussion by danielginovker** [community/community#5221] [Other Feature Feedback, Questions, & Ideas]
Open Source projects are the backbone to everything.

We're finally starting to see a world where large corporations realize that funding open source gives everyone an advantage - yet, Github, a platform where this could all happen, wants to take **10%** of these (potentially massive) donations.

Yes, Github should make money. But this is an absurd cut. I'm trying to talk to my company about sponsoring projects, but they see it as "let's see if we can do something more direct instead of giving Microsoft 1/10th of our donation", then nothing happens. (Paraphrasing, I'm aware there's no 10% in beta, but recurring will overlap into it etc etc etc).

Github- Sponsors is a good thing. It's great when the little guy does it too. But please reconsider the fee.

**Comment by songololo** [comment]
Absolutely agree. The 10% is a show-stopper especially for higher priced categories.
